### PR TITLE
Switch oss and knative from plank to prow-controller-manager

### PR DESCRIPTION
**Pull request by fejta**
/hold
/assign @chaodaiG @chases2

### CODE REVIEW

File: prow/oss/cluster/prow-controller-manager.yaml [hunk 1]
```diff
@@ -22,32 +21,32 @@ spec:
       - name: prow-controller-manager
         image: gcr.io/k8s-prow/prow-controller-manager:v20201026-f7d653694b
         args:
-        - --config-path=/etc/config/config.yaml
+        - --config-path=/etc/config/config-20201002
```
Comment (chaodaiG) [inline prow/oss/cluster/prow-controller-manager.yaml:24]: I think the line needs `20201002` is `- --kubeconfig=/etc/kubeconfig/config-20201002`, which is line # 28

Comment (fejta) [reply prow/oss/cluster/prow-controller-manager.yaml:24]: Done

Comment (fejta) [reply prow/oss/cluster/prow-controller-manager.yaml:24]: Ugh, no, one sec

Comment (fejta) [reply prow/oss/cluster/prow-controller-manager.yaml]: Done

**Pull request merged**